figures/models.py

```diff
@@ -5,6 +5,7 @@
 
 from __future__ import absolute_import
 from datetime import date
+import timeit
```

Comment:
Do you `timeit` safe for production use? I don't know. I've checked https://docs.python.org/3/library/timeit.html and it shows a GCP related note:

> Note By default, [timeit()](https://docs.python.org/3/library/timeit.html#timeit.Timer.timeit) temporarily turns off [garbage collection](https://docs.python.org/3/glossary.html#term-garbage-collection) during the timing. The advantage of this approach is that it makes independent timings more comparable. The disadvantage is that GC may be an important component of the performance of the function being measured. If so, GC can be re-enabled as the first statement in the setup string. For example

I think it's only related to the following usage `timeit.Timer('for i in range(10): oct(i)', 'gc.enable()').timeit()`.

I think it's safe, but want to double check if we've missed something here.

Reply:
Thank you Omar! To be on the safe side, I'll just capture start and end times without timeit